src/Native/System.Native/pal_console.cpp

```diff
@@ -458,7 +458,9 @@ static bool InitializeSignalHandling()
     }
 
     // Finally, register our signal handlers
-    struct sigaction newAction = { .sa_flags = SA_RESTART | SA_SIGINFO };
+    struct sigaction newAction;
```

Comment:
What about the rest of the fields?  Aren't they going to be left as garbage?  If we can't use the ```= { .field = value};``` syntax, we should do something else to zero out the struct first.  Can we do ```= {0};```?  Or if not, memset it to zero?